rankee.py

```diff
@@ -0,0 +1,56 @@
+from collections import defaultdict
+
+from peewee import *
+
+from models import *
+from elo import rank_teams
+
+class Rankee:
```

Comment:
why this and `Player`...?

Reply:
`Player` represents someone losing or winning a single game. `Rankee` is what your old `Player` was and is used to decouple tracking ranks from the game records.